setup-container.sh

```diff
@@ -331,6 +342,36 @@ function parse_arguments() {
     fi
 }
 
+function start_existing_container() {
+    if [ -z ${IMAGE_ID} ]
+    then
+        echo "Missing required argument -i IMAGE_ID to start the container"
+        exit 1
+    fi
+    container_id=`docker container ls --all --filter=ancestor=${IMAGE_ID} --format '{{.ID}}'`
+    count=`echo $container_id | wc -w`
+    if [ $count -eq 0 ]
+    then
+        echo "No containers found for image ${IMAGE_ID}"
+        exit 0
+    fi
+    if [ $count -eq 1 ]
+    then
+        container_name=`docker inspect $container_id --format '{{.Name}}'`
+        CONTAINER_NAME=${container_name}
+        if [[ $container_name == /* ]]
+        then
+            CONTAINER_NAME="${container_name:1}"
+        fi
+        start_local_container ${START_EXISTING_CONTAINER}
```

Comment:
With `-s` option, it's possible that this branch would be hit. But the problem is that after local container is started, there is no `exit`. The script will keep running from line 422, right?

Reply:
@wangxin yes that's right. It would run `show_local_container_login` to display the SSH information.